react/features/base/lib-jitsi-meet/functions.js

```diff
@@ -148,5 +150,5 @@ export function loadConfig(url: string) {
         return value;
     });
 
-    return promise;
+    return timeoutPromise(promise, timeoutMs);
```

Comment:
Actually, the problem that we're trying to work around with the timeout is the network request. What's being timed out here is not only the network request - `loadScript` - but also the subsequent processing which we technically didn't really set out to time out.